nodes/sample_host.json

```diff
@@ -39,10 +39,9 @@
       "domain_names": ["<domain name>", "<domain name>", "<...>"],
       "redirect_domain_names": ["<domain name>", "<domain name>", "<...>"],
       "ruby_version": "2.1.0",
-      "ssl_info": {
-        "key": "<ssl key>",
-        "crt": "<ssl crt>"
-        },
+      "ssl_enabled": true,
```

Comment:
What do you think about adding this information back into a key value hash like this:

```
"ssl_info" : {
  "enabled": true
  "certificate": "The cert file",
  "certificate_key": "The key for the cert file"
}
```